### ISSUE TITLE
aria-controls relation should be abstracted

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/john-travoltage#184]
Related to #181 and #180 and #134 and #171 - When AriaHerald is generalized, it should also be able to set up the aria-controls relationship (see https://www.w3.org/TR/wai-aria/states_and_properties#aria-controls for behavior).

**Comment by jessegreenberg** [comment]
Done above, closing.
